app/design/frontend/base/default/template/boltpay/track.phtml

```diff
@@ -34,7 +34,7 @@ $version = (string)$versionElm[0];
             id="bolt-track"
             type="text/javascript"
             src="<?= $this->getTrackJsUrl(); ?>"
-            data-publishable-key="<?=$this->getPublishableKey(); ?>">
+            data-publishable-key="<?=$this->getPublishableKey('multi-page'); ?>">
```

Comment:
We can live with this for now but ideally this should be either one-page or multi-page, depending on which one is set.

Reply:
Sure.  It has always been the way it is now.  I was only going through the code and explicitly writing out implicit parameters.  No need to wait.  We can implement that now.  What it means is that in production, there are probably some cases where people only have a payment only key and their track.js is not working properly.